src/objects/media/media.stories.mdx

```diff
@@ -21,19 +21,19 @@ const exampleText =
 // respond to args more dynamically than other demos.
 const imageDemoTransformSource = (_src, storyContext) => {
   const { args } = storyContext;
-  const withStr = args.reverse
+  const withStr = args?.reverse
```

Comment:
This was causing the Docs page for the Media object to fail to render.

Reply:
Thank you, this seems to be happening on multiple components and objects, I’ve noticed.